hr_expense_cancel/models/account_payment.py

```diff
@@ -8,6 +8,4 @@ class AccountPayment(models.Model):
     _inherit = "account.payment"
 
     expense_sheet_id = fields.Many2one(
-        comodel_name="hr.expense.sheet",
-        string="Expense sheet",
-    )
+        "hr.expense.sheet", string="Expense sheet")
```

Comment:
Why do you change this without need?

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)